src/common/blkdev.cc

```diff
@@ -9,29 +11,26 @@
  * Foundation.  See file COPYING.
  * 
  */
-
 #include <errno.h>
 #include <sys/ioctl.h>
 #include <sys/stat.h>
 #include <dirent.h>
 #include "include/uuid.h"
 #include "blkdev.h"
 
-
 #ifdef __linux__
 #include <linux/fs.h>
 #include <blkid/blkid.h>
 
+#ifndef UUID_LEN
 #define UUID_LEN 36
+#endif
 
 static const char *sandbox_dir = "";
 
 void set_block_device_sandbox_dir(const char *dir)
 {
-  if (dir)
-    sandbox_dir = dir;
-  else
-    sandbox_dir = "";
+  dir ? sandbox_dir = dir : sandbox_dir = "";
```

Comment:
@shinobu-x 
Don't really like the "hackish" way of trying to do the same as the it-then-else.
Would just leave it as is.